
 This patch causes a message to be printed if the migration job finds that
 the target job has already been migrated:

    JobId %s already migrated probably by another Job. Migration stopped.

  It can be applied to 2.4.2 (not to previous versions) with:

  cd <bacula-source>
  patch -p0 <2.4.2-mig-message.patch
  ./configure <your-options>
  make
  ...
  make install


Index: src/dird/migrate.c
===================================================================
--- src/dird/migrate.c	(revision 7444)
+++ src/dird/migrate.c	(working copy)
@@ -284,6 +284,8 @@
    }
    /* Make sure this job was not already migrated */
    if (jcr->previous_jr.JobType != JT_BACKUP) {
+      Jmsg(jcr, M_INFO, 0, _("JobId %s already migrated probably by another Job. Migration stopped.\n"),
+         edit_int64(jcr->previous_jr.JobId, ed1));
       set_jcr_job_status(jcr, JS_Terminated);
       migration_cleanup(jcr, jcr->JobStatus);
       return true;
